.bazelrc

```diff
@@ -58,6 +58,7 @@ test --test_verbose_timeout_warnings
 
 # Allow tags to influence execution requirements
 common --experimental_allow_tags_propagation
+common --noenable_bzlmod
```

Comment:
@phlax ,
It doesn't seams to apply during ci . Any idea where it would be appropriate to apply it ?

Reply:
it should work - its the correct place to do this - which ci?

Reply:
also - im thinking it should be disabled by default in our current bazel version

Reply:
> it should work - its the correct place to do this - which ci?

That's what is advised by the guide : https://bazel.build/external/migration#enable-bzlmod

Reply:
> also - im thinking it should be disabled by default in our current bazel version

I think so but depending on bazel version, it's changing

Reply:
> which ci? 

The azure pipeline . Bazelist is by default using the last version of bazel which is 7.0.2 . In that version bzlmod is enabled by default.
what I could see in the pipeline is  a --norc argument which can make the config not being applied for that stage

Reply:
it should be reading `.bazelversion` - do you have a link?

Reply:
also not seeing `norc` with `git grep` so wondering where that could come from

Reply:
See https://dev.azure.com/cncf/envoy/_build/results?buildId=163900&view=logs&j=c7de00aa-b0fb-52ba-707b-da6afd918eda&t=449299ab-9da3-5c6b-afe3-e0c5dbf1dfd1

> /usr/bin/bash --noprofile --norc /home/vsts/work/_temp/272decbb-aa73-4860-9316-a0f8feefd019.sh

Reply:
i see - i think that is unrelated - its an argument to bash not to read (eg) rc files in home dir

Reply:
the bazel version is 6.5.0 on that ci https://source.cloud.google.com/results/invocations/cd77d311-709d-4713-8fd5-c8b2f93c8ce4/details